rasa/utils/plotting.py

```diff
@@ -116,15 +116,15 @@ def plot_confusion_matrix(
         fig.savefig(output_file, bbox_inches="tight")
 
 
-def plot_histogram(
+def plot_comparative_histogram(
```

Comment:
Please preserve the original method name (`plot_histogram`) for backwards compatibility

Reply:
Done